microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/activation/AnnotationActivationCompatibilityTest.java

```diff
@@ -24,16 +24,18 @@
 import io.restassured.RestAssured;
 
 /**
- * Just verify a raw token value can be injected into a variable in an application class a compare it has expected value
+ * Set of test which are verifying that {@code @LoginConfig} annotation works as it should - which activates the JWT
```

Comment:
`Set of test` -> `Set of tests`
`... - which activates...` -> `... - i.e. activating`

Reply:
Done, thanks